packages/opentelemetry-plugin-mongodb/package.json

```diff
@@ -53,7 +56,8 @@
   },
   "dependencies": {
     "@opentelemetry/core": "^0.1.0",
+    "@opentelemetry/types": "^0.1.0",
     "@opentelemetry/node": "^0.1.0",
```

Comment:
should not be in deps, devdeps maybe ?

Reply:
Do you need it for any enums? Those are runtime deps, whereas interfaces are not.

Reply:
The `types` are theorically not needed but TS still output a `require('@opentelemetry/types')` inside the JS so i would prefer to not move it. However `node` is only needed in dev you are right, i will move it.